nerf/renderer.py

```diff
@@ -13,6 +13,7 @@
 
 import raymarching
 import nvdiffrast.torch as dr
+glctx = dr.RasterizeGLContext(output_db=False)
```

Comment:
why is this needed?

Reply:
Found that declaring gl context will crash without defining right after importing dr when I'm using command line mode. Seems that it need to be defined on top of all other classes and object

Reply:
I think it may be related to environmental issues on your side since I don't meet this problem. Also, `glctx` is never used (we use `self.glctx`)